pkg/k8s/configmap.go

```diff
@@ -14,6 +17,7 @@ func ConfigMap(name, namespace, filename, data string) *corev1.ConfigMap {
 		ObjectMeta: metav1.ObjectMeta{
 			Name:      name,
 			Namespace: namespace,
+			Labels:    naming.Labels(name, cr.Name, "percona-server", "database"),
```

Comment:
Given that we start version checking, I think we should check the version here and then assign the labels, right?